src/string_map.c

```diff
@@ -111,8 +111,8 @@ rcutils_string_map_get_capacity(const rcutils_string_map_t * string_map, size_t
     string_map->impl, "invalid string map",
     return RCUTILS_RET_STRING_MAP_INVALID, rcutils_get_default_allocator())
   RCUTILS_CHECK_ARGUMENT_FOR_NULL(
-    capacity, RCUTILS_RET_INVALID_ARGUMENT, rcutils_get_default_allocator())
-  *capacity = string_map->impl->capacity;
+    capacity, RCUTILS_RET_INVALID_ARGUMENT, rcutils_get_default_allocator()) *
+  capacity = string_map->impl->capacity;
```

Comment:
Thanks I didn't notice that when reviewing what it did. Unfortunately the comment didn't help, it still though it was part of the previous statement and put a space between `*` and `capacity`. I also couldn't put a semicolon on the macro because that would trigger the unnecessary semicolon warning from the compiler.

So I just turned uncrustify off on that line: 38cbbca